README.md

```diff
@@ -1,2 +1,25 @@
-# hw01
```

Comment:
Evaluation HW-1 for Maurice Bokanga
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     √      |              |            |
| **Coding strategy**         |     √      |              |            |
| **Presentation**            |    √       |              |            |
| **Achievement, creativity** |     √      |              |            |
| **Ease of access**          |    √       |              |            |

As this is the first assignment, there are not many comments I could make on this one.
The code was written in a proper way and it is quite interesting to see the bear appear before everything.
Overall it is a well-written code and obviously reached the given requirement.